apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,111 +18,172 @@ export function InstructorCourseAdminSettings({
       </head>
       <body>
         ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", { ...resLocals })}
-        <main id="content" class="container-fluid">
+        <main id="content" class="container mb-5 px-0">
```

Comment:
Why `px-0` here? This gets rid of horizontal padding on narrow viewports.

I like where you're going with `mb-5`, but conventionally on other pages, we've used `mb-4` applied to the card, e.g.:

- https://github.com/PrairieLearn/PrairieLearn/blob/aefa9184d691c78db7064abdcfdb9d7db04873a5/apps/prairielearn/src/pages/instructorCourseAdminSets/instructorCourseAdminSets.ejs#L10
- https://github.com/PrairieLearn/PrairieLearn/blob/aefa9184d691c78db7064abdcfdb9d7db04873a5/apps/prairielearn/src/pages/instructorCourseAdminTags/instructorCourseAdminTags.ejs#L10